test/Parallax.spec.js

```diff
@@ -36,11 +36,28 @@ describe('<Parallax />', () => {
       expect(parallaxInitMock).toHaveBeenCalledTimes(1);
     });
 
+    test('should have default options if none are given', () => {
+      mount(<Parallax />);
+      expect(parallaxInitMock).toHaveBeenCalledWith({
+        responsiveThreshold: 0
+      });
+    });
+
     test('should call Parallax with the given options', () => {
       const options = { responsiveThreshold: 200 };
 
       mount(<Parallax options={options} />);
       expect(parallaxInitMock).toHaveBeenCalledWith(options);
     });
+
+    test('can render children element', () => {
```

Comment:
how about we add the required prop and use to match snapshot here?

Reply:
@alextrastero please, have a look whenever you can :grinning: